.github/workflows/pr.yml

```diff
@@ -84,6 +84,12 @@ jobs:
             git --no-pager diff ':(exclude)bin/kustomize' ':(exclude)bin/controller-gen'
             exit 1
           fi
+      - name: Run Gosec Security Scanner
+        uses: securego/gosec@master
+        with:
+          args: ./...
```

Comment:
Do we want to scan tests files? We have some hits there.

Reply:
I would default to say yes, test everything, but honestly I have no strong feeling either way.
What is your preference @davidmogar ?

Reply:
I'd say yes skipping generated files.

Reply:
Done. I found an option to `gosec` to ignore generated files.
```
$ ~/go/bin/gosec -exclude-generated ./...
...
[gosec] 2022/08/16 12:07:48 Checking file: /home/jdisnard/scm/github/forks/release-service/api/v1alpha1/releasestrategy_types.go
[gosec] 2022/08/16 12:07:48 Ignoring generated file: /home/jdisnard/scm/github/forks/release-service/api/v1alpha1/zz_generated.deepcopy.go
[gosec] 2022/08/16 12:07:48 Checking package: tekton
...
```
It works as expected, but one note is the option is not well documented:
https://github.com/securego/gosec/issues/709